### ISSUE TITLE
Cannot redefine property: babelPluginFlowReactPropTypes_proptype_Action

### ISSUE THREAD

**Issue by haf** [brigand/babel-plugin-flow-react-proptypes#148]
```
/actions.js:34
  Object.defineProperty(exports, key, {
         ^

TypeError: Cannot redefine property: babelPluginFlowReactPropTypes_proptype_Action
    at Function.defineProperty (native)
```

When I upgraded this lib from 5.1.2 to 9.1.1

It doesn't say what line of code I have to change, and I can't change all of them.

**Comment by brigand** [comment]
@haf can you show the file this occurs in, and run `npx babel src/whatever.js` and show the output?

**Comment by haf** [comment]
It happens across the largest batch of files in the project; I expect it's because every file is exported with star.

```
export * from './CompA/actions.js';
export * from './CompB/actions.js';
```

and each actions has


```
type Action
  | ACtionA
  | ActionB
```

But since it's across so many proprietary files, I can't be certain, especially as the exports are not actually used by react components.

**Comment by brigand** [comment]
Thanks for the info. I think the issue is that the two actions.js files are exporting a type with the same name. Is that the case?

This may have broken it: https://github.com/brigand/babel-plugin-flow-react-proptypes/pull/141

**Comment by haf** [comment]
Yes, they all export the same `type Action`.

**Comment by brigand** [comment]
Reverted #141 and published as 10.0.0. Let me know if it works out.

**Comment by haf** [comment]
Works perfectly! Thank you!
